cmd/deploy/start.go

```diff
@@ -82,6 +84,7 @@ func NewDeployStartCmd(configuration *config.Configuration) *cobra.Command {
 			return start(cmd, configuration, options)
 		},
 	}
+	cmd.Flags().BoolVarP(&options.dryRun, "dryRun", "d", false, "output the json of the deployment request without submitting it")
```

Comment:
not a big deal but it would be nice to have all of the flags as constants so that we don't refer to them by magic strings everywhere